app/views/admin/conferences/custom_domain.html.haml

```diff
@@ -0,0 +1,82 @@
+.row
+  .col-md-12
+    %h1
+      Custom domain
+    %p.text-muted
+      Point your own domain to #{@conference.title}
+%hr
+.row
+  .col-md-12
+    %table.table
+      %thead
+        %th Domain Name
+        %th Correctly points to #{@conference.short_title}
+        %th Actions
+      %tbody
+        %tr
+          %td
+            = @conference.custom_domain
+          %td
+            - ck_domain = @conference.check_custom_domain
+            - if ck_domain == true
+              %span.glyphicon.glyphicon-ok
+            - elsif ck_domain == false
+              %span.glyphicon.glyphicon-remove
+            - else
+              %i
+                = ck_domain
+          %td
+            .btn-group
+              = link_to 'Edit', attach_custom_domain_admin_conference_path(@conference.short_title),
+                method: :get, class: 'btn btn-primary'
+              = link_to 'Delete', admin_conference_path(@conference.short_title),
+                method: :delete, class: 'btn btn-danger'
+.row
+  .col-md-9.text-right
+    = link_to '#status-help', class: 'btn btn-default',  "data-toggle"=>"collapse" do
+      Help?
```

Comment:
Either ask "Do you need help?" with the question mark, or just say "Help" with no question mark, but "Help?" looks funny :P